### DISCUSSION TITLE
High-performance GitHub-hosted runners Static IP

### DISCUSSION THREAD

**Discussion by randyshoopman** [community/community#41031] [Actions]
### High-performance GitHub-hosted runners Beta

Product Feedback

### Body

I was excited to hear about GitHub's new high-performance GitHub-hosted runners and joined the beta.   Unfortunately I was disappointed to find that we cannot use static IPs because we are not an Enterprise account.  Static IPs would allow us to set firewall rules on other machines needed to run Actions.   Trying to set firewall rules for the numerous dynamic IPs that can change at anytime is a pain.  Not having static IPs means we'll probably just go back to self hosted runners.

Why not make this feature available to Team accounts and just bill more for usage?  Please consider making the static IP feature available to Team accounts.  Thanks

**Comment by ossanna16** [comment]
Hi there @randyshoopman and welcome to our community! Thank you for the feedback!
